torch/package/package_exporter.py

```diff
@@ -39,6 +42,13 @@
 
 ActionHook = Callable[["PackageExporter", str], None]
 
+# list of modules which selectively interned by default. These are generally unstable
+#  modules which should not be externed due to lack of backwards compatibility
+DEFAULT_SELECTIVE_INTERN_LIST = {}
+
+DEFAULT_SELECTIVE_INTERN_LIST = {"torch":
```

Comment:
This introduces a new place where we need to look when matching modules against actions. In general, we should try to avoid introducing new sources of state to keep complexity down. Can we express this as a set of patterns that are added to every PackageExporter instead?

Reply:
Wait that is what I was trying to do here, do you have any recommendations to make things more clear?

Reply:
Sorry actually I put this in the wrong place, I meant `self.selective_interns` vs. the patterns. Right now it's not clear to me why we have both vs. just using patterns to track everything.

Reply:
ahh gotcha, it's to draw a connection between the selective_extern and selective_intern modules, so that we can preload the packages in once a selective_extern module is imported. Alternatively, you have to go through every pattern and check if the prefix matches a selective intern, and doesn't have a wildcard, and do that for every selective extern package. This scales pretty poorly, so I thought using a dict would be cleaner.